### PR TITLE
Synchronization sources from parallel development

### PR DESCRIPTION
**Pull request by KovalM**

### CODE REVIEW

File: ims/ostis_tech/lib_ostis/sectn_lib_of_reusable_comp_ui/ui_menu/activ_auto/classification_agents/ui_menu_file_for_finding_all_drafts_of_author/lib_ui_menu_file_for_finding_all_drafts_of_author.scs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+<- rrel_key_sc_element:
+	... 
+	(*
+	<- explanation;;
+	<= nrel_sc_text_translation:
+		...
+		(*
+		-> rrel_example:
+			"file://htmls/Ui_Of_finding_all_drafts_of_author.html"
```
Comment (deniskoronchik) [inline ims/ostis_tech/lib_ostis/sectn_lib_of_reusable_comp_ui/ui_menu/activ_auto/classification_agents/ui_menu_file_for_finding_all_drafts_of_author/lib_ui_menu_file_for_finding_all_drafts_of_author.scs:20]:
```suggestion
			"file://htmls/ui_of_finding_all_drafts_of_author.html"
```

File: ims/ostis_tech/lib_ostis/sectn_lib_of_reusable_comp_ui/ui_menu/activ_auto/classification_agents/ui_menu_file_for_finding_all_drafts_of_author/ui_menu_file_for_finding_all_drafts_of_author.scsi [hunk 2]
```diff
@@ -0,0 +1,25 @@
+ui_menu_file_for_finding_all_drafts_of_author <- ui_user_command_class_atom; ui_user_command_class_view_kb; ui_one_argument_command_class;;
+
+ui_menu_file_for_finding_all_drafts_of_author => nrel_main_idtf: 
+	[Найти все черновики разработчика] 
+	(* <- lang_ru;; *);;
+
+ui_menu_file_for_finding_all_drafts_of_author => nrel_main_idtf: 
+	[Find all drafts of author]
```
Comment (deniskoronchik) [inline ims/ostis_tech/lib_ostis/sectn_lib_of_reusable_comp_ui/ui_menu/activ_auto/classification_agents/ui_menu_file_for_finding_all_drafts_of_author/ui_menu_file_for_finding_all_drafts_of_author.scsi:8]:
```suggestion
	[Find all drafts by author] 
```

File: ims/ostis_tech/lib_ostis/sectn_lib_of_reusable_comp_ui/ui_menu/activ_auto/classification_agents/ui_menu_file_for_finding_all_drafts_of_question_author/htmls/Ui_Of_finding_all_drafts_of_question_author.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<html>
```
Comment (deniskoronchik) [inline ims/ostis_tech/lib_ostis/sectn_lib_of_reusable_comp_ui/ui_menu/activ_auto/classification_agents/ui_menu_file_for_finding_all_drafts_of_question_author/htmls/Ui_Of_finding_all_drafts_of_question_author.html:1]: File name

File: ims/ostis_tech/lib_ostis/sectn_lib_of_reusable_comp_ui/ui_menu/activ_auto/classification_agents/ui_menu_file_for_finding_all_drafts_of_question_author/lib_ui_menu_file_for_finding_all_drafts_of_question_author.scs [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+<- rrel_key_sc_element:
+	... 
+	(*
+	<- explanation;;
+	<= nrel_sc_text_translation:
+		...
+		(*
+		-> rrel_example:
+			"file://htmls/Ui_Of_finding_all_drafts_of_question_author.html"
```
Comment (deniskoronchik) [inline ims/ostis_tech/lib_ostis/sectn_lib_of_reusable_comp_ui/ui_menu/activ_auto/classification_agents/ui_menu_file_for_finding_all_drafts_of_question_author/lib_ui_menu_file_for_finding_all_drafts_of_question_author.scs:20]:
```suggestion
			"file://htmls/ui_of_finding_all_drafts_of_question_author.html"
```

File: ims/ostis_tech/lib_ostis/sectn_lib_of_reusable_comp_ui/ui_menu/activ_auto/classification_agents/ui_menu_file_for_finding_all_proposals_of_question_author/lib_ui_menu_file_for_finding_all_proposals_by_question_author.scs [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+<- rrel_key_sc_element:
+	... 
+	(*
+	<- explanation;;
+	<= nrel_sc_text_translation:
+		...
+		(*
+		-> rrel_example:
+			"file://htmls/Ui_Of_finding_all_proposals_of_question_author.html"
```
Comment (deniskoronchik) [inline ims/ostis_tech/lib_ostis/sectn_lib_of_reusable_comp_ui/ui_menu/activ_auto/classification_agents/ui_menu_file_for_finding_all_proposals_of_question_author/lib_ui_menu_file_for_finding_all_proposals_by_question_author.scs:20]:
```suggestion
			"file://htmls/ui_of_finding_all_proposals_of_question_author.html"
```

File: ims/ostis_tech/lib_ostis/sectn_lib_reusable_comp_kpm/reusable_sc_agents/lib_scp_agents/activity_automatisation/classification_agents/lib_component_agent_of_finding_all_drafts_of_author/lib_component_agent_finding_all_drafts_of_author.scs [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+<- rrel_key_sc_element: 
+	...
+	(*
+	<- explanation;;
+	<= nrel_sc_text_translation:
+		...
+		(*
+		-> rrel_example:
+			"file://htmls/Agent_of_finding_all_drafts_of_author.html"
```
Comment (deniskoronchik) [inline ims/ostis_tech/lib_ostis/sectn_lib_reusable_comp_kpm/reusable_sc_agents/lib_scp_agents/activity_automatisation/classification_agents/lib_component_agent_of_finding_all_drafts_of_author/lib_component_agent_finding_all_drafts_of_author.scs:22]:
```suggestion
			"file://htmls/agent_of_finding_all_drafts_of_author.html"

```

File: ims/ostis_tech/lib_ostis/sectn_lib_reusable_comp_kpm/reusable_sc_agents/lib_scp_agents/activity_automatisation/classification_agents/lib_component_agent_of_finding_all_drafts_of_question_author/lib_component_agent_of_finding_all_drafts_of_question_author.scs [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+<- rrel_key_sc_element: 
+	...
+	(*
+	<- explanation;;
+	<= nrel_sc_text_translation:
+		...
+		(*
+		-> rrel_example:
+			"file://htmls/Agent_of_finding_all_drafts_of_question_author.html"
```
Comment (deniskoronchik) [inline ims/ostis_tech/lib_ostis/sectn_lib_reusable_comp_kpm/reusable_sc_agents/lib_scp_agents/activity_automatisation/classification_agents/lib_component_agent_of_finding_all_drafts_of_question_author/lib_component_agent_of_finding_all_drafts_of_question_author.scs:22]:
```suggestion
			"file://htmls/agent_of_finding_all_drafts_of_question_author.html"

```

File: ims/ostis_tech/lib_ostis/sectn_lib_reusable_comp_kpm/reusable_sc_agents/lib_scp_agents/activity_automatisation/classification_agents/lib_component_agent_of_finding_all_proposals_of_question_author/lib_component_agent_finding_all_proposals_of_question_author.scs [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+
+<- rrel_key_sc_element: 
+	...
+	(*
+	<- explanation;;
+	<= nrel_sc_text_translation:
+		...
+		(*
+		-> rrel_example:
+			"file://htmls/Agent_of_finding_all_proposals_of_question_author.html"
```
Comment (deniskoronchik) [inline ims/ostis_tech/lib_ostis/sectn_lib_reusable_comp_kpm/reusable_sc_agents/lib_scp_agents/activity_automatisation/classification_agents/lib_component_agent_of_finding_all_proposals_of_question_author/lib_component_agent_finding_all_proposals_of_question_author.scs:22]:
```suggestion
			"file://htmls/agent_of_finding_all_proposals_of_question_author.html"

```

**Review by deniskoronchik — COMMENTED** [review]
Small fixes

**Review by deniskoronchik — APPROVED** [review]

**Review by MikhailSadovsky — APPROVED** [review]

**Review by ShunkevichDV — APPROVED** [review]

**Pull request merged**